react/features/conference/components/web/Subject.js

```diff
@@ -61,9 +63,10 @@ class Subject extends Component<Props> {
  */
 function _mapStateToProps(state) {
     const { subject } = state['features/base/conference'];
+    const conferenceName = getConferenceName(state);
 
     return {
-        _subject: subject,
+        _subject: subject || conferenceName,
```

Comment:
`getConferenceName` already checks for the subject first, so there is no need to do that here.